test/integration/cluster/cluster_test.go

```diff
@@ -28,13 +28,13 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/clientcmd"
-	clusterv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	clusterv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha2"
```

Comment:
Not sure if you want to do it in this PR (although it would probably make sense), but anywhere we have the alias `clusterv1alpha1`, it should be changed to `clusterv1alpha2`.

Reply:
Done